docs/HowTo/Deploy/Monitoring-Performance.md

```diff
@@ -71,9 +71,23 @@ block of the `prometheus.yml` file:
 1. Choose **Graph** from the menu bar and click the **Console** tab below.
 
 1. From the **Insert metric at cursor** drop-down, select a metric such as `besu_blockchain_difficulty_total` or
-`besu_blockchain_height` and click **Execute**. The values are displayed below.
+`besu_blockchain_height` and click **Execute**. The values are displayed.
 
-    Click the **Graph** tab to view the data as a time-based graph. The query string is displayed below the graph. 
+    The following table shows metrics that Besu might capture and share with Prometheus.
+    
+    | Name | Metric Type | Definition | JSON-RPC Equivalent |
+    | ---  | ---         | ---        | ---                 |                   
+    | ethereum_blockchain_height | Gauge | The current height of the canonical chain | `eth_blockNumber` |
+    | ethereum_best_known_block_number | Gauge | The estimated highest block available | `highestBlock` of `eth_syncing` or `eth_blockNumber`, if not syncing |
```

Comment:
The estimated highest block available -> Estimated highest block available

Suggestion: Would it be clearer if _if not syncing_ was in brackets?